python/py-ezdxf/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup python        1.0
+PortGroup github        1.0
+
+name                    py-ezdxf
+version                 1.2.0
+revision                0
+
+github.setup            mozman ezdxf ${version} v
+
+checksums               rmd160  70f9c4204aa57ae9bc893e607c4a460689acfb82 \
+                        sha256  7c4476fc715fc23d813755ac5e88a4169b2d87218d239a671223653db1301369 \
+                        size    43986058
+
+maintainers             @flyingsamson openmaintainer
+description             Python interface to DXF
+long_description        This Python package is designed to facilitate the creation and manipulation \
+                        of DXF documents, with compatibility across various DXF versions. It empowers \
+                        users to seamlessly load and edit DXF files while preserving all content, \
+                        except for comments.
+license                 MIT
+
+homepage                https://ezdxf.mozman.at
+
+python.versions         38 39 310 311 312
+
+github.livecheck.regex  {([0-9.]+)}
+
+if {${name} ne ${subport}} {
+    depends_build       port:py${python.version}-setuptools
```

Comment:
For subports 37 and above the setuptools PEP517 backend will be used. So no need to add this dependency explicitly here.

Reply:
Removed